spec/javascript/e2e/SupplementalApplicationPage/confirmed_household.e2e.js

```diff
@@ -0,0 +1,46 @@
+import puppeteer from 'puppeteer'
```

Comment:
I broke up lease and confirmed HH income tests to help with clearness of testing, but it does lead to more overhead with login and navigation to the page. thoughts on whether we should try to combine them or keep them separated?

Reply:
would be nice to not have that additional overhead. perhaps we can situate both within the same test, but break the steps for checking lease fields and the code for checking HH income into separate files, to keep some separation and clarity?

Reply:
I think the issue with that would be that we still wouldn't get a helpful error message if one of the tests failed

Reply:
could we use unit testing here? would that get at what we're trying to test in this scenario?

Reply:
unfortunately I don't think so because we don't have any other E2E integration testing for the supplemental application yet and the stripping of the currency values happens in the mapping step

Reply:
update on this conversation: @akegan and I discussed on a call and decided to let these tests be but also to cap the number of separate e2e test areas for the supp app to the current 5